src/components/donation-flow/RadioAccordionGroup.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import {
+  Box,
+  BoxProps,
+  Button,
+  Collapse,
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  RadioGroupProps,
+  TextField,
+} from '@mui/material'
+import { styled } from '@mui/material/styles'
+import theme from 'common/theme'
+import CardIcon from './CardIcon'
+import BankIcon from './BankIcon'
+
+export const StyledRadioAccordionItem = styled(Box)(() => ({
+  '&:not(:last-child)': {
+    borderBottom: `1px solid ${theme.borders.dark}`,
+  },
+  padding: theme.spacing(2),
+  margin: 0,
+  cursor: 'pointer',
+}))
+
+interface RadioAccordionItemProps extends BoxProps {
+  control: React.ReactNode
+  icon: React.ReactNode
+  content?: React.ReactNode
+  selected?: boolean
+}
+
+// Temporarily here for testing until the components starts being used
+export const testRadioOptions: Option[] = [
+  {
+    value: 'card',
+    label: 'Card',
+    content: (
+      <div style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
You can also use `<Stack direction="column">` for an easy flex component that can be styled via `sx` prop.

Reply:
> @dimitur2204 Nice addition!
> 
> I've checked with the designs, and the new component with payment types is used in the app's mobile version.
> 
> In the desktop version, I see three horizontal buttons for the donation + radio select group for the registration method.
> 
> In both scenarios the new component is used, so we're good to go 🚀
> 
> Desktop	Mobile
> ![image](https://user-images.githubusercontent.com/893608/210593636-45269a92-23fc-443b-a3bb-b8d02710b5f5.png)	![image](https://user-images.githubusercontent.com/893608/210594608-747d84f4-27c9-45f3-b571-3c501edd6d0d.png)

In that case for the desktop version probably creating a new component will be easier, because of how different it is.

Reply:
This is there just temporarily for testing purposes, so I will merge it with it, but a good catch either way